sno/working_copy/gpkg.py

```diff
@@ -639,9 +639,10 @@ def write_full(self, target_tree_or_commit, *datasets, safe=True):
 
                 CHUNK_SIZE = 10000
                 total_features = dataset.feature_count
-                for rows in self._chunk(dataset.feature_tuples(col_names), CHUNK_SIZE):
-                    dbcur.executemany(sql_insert_features, rows)
-                    feat_progress += len(rows)
+                for row_dicts in self._chunk(dataset.features(), CHUNK_SIZE):
+                    row_tuples = (row_dict.values() for row_dict in row_dicts)
+                    dbcur.executemany(sql_insert_features, row_tuples)
+                    feat_progress += db.changes()  # DONOTSUBMIT
```

Comment:
☝️ 

I can't think of a reason why this would be different than `len(row_dicts)`, since `INSERT INTO` always inserts a row

Reply:
Done